cl/users/urls.py

```diff
@@ -94,11 +99,24 @@
         name="view_deleted_visualizations",
     ),
     path("profile/api/", views.view_api, name="view_api"),
+    path("profile/api-token/", views.view_api_token, name="view_api_token"),
+    path("profile/api-usage/", views.view_api_usage, name="view_api_usage"),
+    path("profile/webhooks/", views.view_webhooks, name="view_webhooks"),
+    path(
+        "profile/auto_subscribe/toggle/",
+        views.toggle_recap_email_auto_subscription,
+        name="toggle_recap_email_auto_subscription",
+    ),
     path(
         "profile/password/change/",
         views.password_change,
         name="password_change",
     ),
+    path(
+        "profile/recap/",
```

Comment:
I think we can put periods in our URLS, right? So:

```suggestion
        "profile/recap.email/",
```
If that doesn't work, then:

```suggestion
        "profile/recap-dot-email/",
```

Reply:
Changed.